nncf/openvino/quantization/quantize_model_if_op.py

```diff
@@ -0,0 +1,226 @@
+from itertools import islice
+from pathlib import Path
+from typing import Callable, Dict, List, Optional, Set, Tuple, TypeVar
+
+import openvino.runtime as ov
+from tqdm import tqdm
+
+from nncf import Dataset
+from nncf.common import factory
+from nncf.common.engine import Engine
+from nncf.common.factory import NNCFGraphFactory
+from nncf.common.graph.graph import NNCFGraph
+from nncf.common.graph.graph import NNCFNode
+from nncf.common.graph.model_transformer import ModelTransformer
+from nncf.common.graph.transformations.commands import TargetType
+from nncf.common.graph.transformations.layout import TransformationLayout
+from nncf.common.logging import nncf_logger
+from nncf.common.tensor_statistics.statistic_point import StatisticPointsContainer
+from nncf.openvino.graph.metatypes.openvino_metatypes import OVIfMetatype
+from nncf.openvino.graph.node_utils import has_if_op
+from nncf.openvino.graph.transformations.commands import OVExtractIfSubgraphCommand
+from nncf.openvino.graph.transformations.commands import OVOutputInsertionCommand
+from nncf.openvino.graph.transformations.commands import OVTargetPoint
+from nncf.openvino.graph.transformations.commands import OVUpdateIfSubgraphCommand
+from nncf.quantization.algorithms.algorithm import Algorithm
+
+
+def _make_dataset_for_children_model(
+    engine: Engine,
+    calibration_dataset: Dataset,
+    if_cond_input_name: str,
+    then_model_input_names,
+    else_model_input_names,
```

Comment:
Minor. Missed type hints and descriptions as well.

Reply:
Done